.settings.dev.toml

```
@@ -0,0 +1,35 @@
+[db]
+uri = "postgres://rumba:rumba@127.0.0.1:5432/mdn"
+
+[server]
+host = "0.0.0.0"
```

```suggestion
host = "localhost"
```
let's not encourage people to run this exposed to the network :)